packages/platform/src/editor/Editor.tsx

```diff
@@ -66,7 +71,10 @@ function Placeholder(): JSX.Element {
  * Scripture Editor for USJ. Created for use in Platform.Bible.
  * @see https://github.com/usfm-bible/tcdocs/blob/usj/grammar/usj.js
  *
- * @param props.usj - USJ Scripture data.
+ * @param props.defaultUsj - Default USJ Scripture data.
```

Comment:
Does putting all these `param` comments in jsdocs on the object types themselves carry the comments around everywhere? Seems it might be nice to have the comments on the types so the comments are with the properties. Of course, feel free to ignore this. Just a thought for your consideration.

Reply:
Since `Editor.tsx` is the main component that is exported in the npm package, I wanted the usage information to come up for the developer when they hover over the component. That doesn't happen when the JSDoc is only on the types.

Reply:
Oh... I wonder if we should fix this in Platform.Bible too then 😬